chatgpt-shell.el

```diff
@@ -364,12 +373,13 @@ Uses the interface provided by `comint-mode'"
 
 If region is active, append to prompt."
   (interactive)
-  (let ((prompt (read-string (concat
+  (let ((prompt (completing-read (concat
```

Comment:
Consider keeping read-string and passing chatgpt-shell-default-prompts as history. When the prompt comes up, C-r would give you a similar behaviour to completing-read.

Reply:
With `read-string` I can't get that `C-r` to work, but maybe it's due to something else in my setup. (I can pass `chatgpt-shell-default-prompts` as the `DEFAULT-VALUE` arg which puts them into the future history).

Under vertico completing-read is pretty nice since it's visual, but I'm not sure about what it'd be like with the default completion engine, so I agree perhaps it's not the normal usage of "completion".